sequana/bamtools.py

```diff
@@ -157,6 +156,11 @@ def __init__(self, filename, mode="r", *args):
         self._N = None
         self.reset()
 
+        # we can accumulate the length of each contig/chromosome as an alias
+        bam = pysam.AlignmentFile(self._filename, mode=self._mode, *self._args)
+        hd = bam.header
+        self.lengths = dict([(r,l) for r,l in zip(hd.references, hd.lengths)])
```

Comment:
C'est pas plus performant de directement faire un dict comprehension ?
`{r: l for r,l in zip(hd.references, hd.lengths)}`

Reply:
ok